controllers/pod_client.go

```diff
@@ -312,11 +312,14 @@ func UpdateDynamicFiles(client FdbPodClient, filename string, contents string, u
 		if err != nil {
 			return false, err
 		}
-
 		// We check this more or less instantly, maybe we should add some delay?
 		match, err = client.CheckHash(filename, contents)
 		if !match {
-			log.Info("Waiting for config update", "namespace", client.GetPod().Namespace, "pod", client.GetPod().Name, "file", filename)
+			log.Info("Waiting for config update",
+				"namespace", client.GetPod().Namespace,
+				"cluster", client.GetPod().Labels[fdbtypes.FDBClusterLabel],
```

Comment:
I think that should be fine, the only case where we would hit an issue would be when the Pod doesn't have any label (nil pointer exception). I can also make the cluster optional since for backup and restore Pods we might don't have that label?

Reply:
Should make use of `GetCluster` instead of `GetPod`.